packages/api/test/storage.test.ts

```diff
@@ -100,6 +102,28 @@ describe('storage', () => {
     });
   });
 
+  describe('#getLockfile', () => {
+    it('should retrieve a lockfiles that match our schema', async () => {
+      fetchMock.get('https://dash.readme.com/api/v1/api-registry/n6kvf10vakpemvplx', petstoreSimple);
+
+      // `ajv` has funky types in ESM environments. https://github.com/ajv-validator/ajv/issues/2047
+      // eslint-disable-next-line new-cap
+      const ajv = new Ajv.default();
```

Comment:
Can fix this by setting `moduleResolution` in `test/tsconfig.json` to `Bundler` but because our `module` in the root is `NodeNext` they then become incompatible and TS throws errors. I'm fine leaving this `.default()` quirk in here.